inf-clojure.el

```diff
@@ -208,6 +209,21 @@ whichever process buffer you want to use.")
 
 (put 'inf-clojure-mode 'mode-class 'special)
 
+(defcustom inf-clojure-repl-flavor 'clojure
```

Comment:
Ok I was reading a bit more. By `defcustom` with the types you mean a `defcustom` that uses `:set` to set the local var? The only thing I am missing is how to set the buffer local from a `defcustom`. I am checking also `cider`.